pypact/input/inputdata.py

```diff
@@ -195,13 +215,39 @@ def setDensity(self, densityInGPCC):
 
     def setMass(self, totalMassInKg):
         """
+            Sets the material mode to Mass
             Sets the total mass of the target in kg
             totalMassInKg: mass in kg, must be positive
         """
         if not totalMassInKg > 0.0:
             raise PypactOutOfRangeException("Total mass must be positive.")
 
         self.inventorymass.totalMass = totalMassInKg
+        self.inventoryismass = True
+        self.inventoryisfuel = False
+
+    def setFuel(self):
+        """
+            Sets the material mode to Fuel
+        """
+
+        self.inventoryismass = False
+        self.inventoryisfuel = True
+
+    def addIsotope(self, isotope, numberOfAtoms):
+        """
+            Add an isotope
+            isotope: character symbol of element name, e.g. 'Fe' and the mass number of the isotpe e.g '56'
+            numberOfAtoms: the number of atoms present 
+        """
+        if numberOfAtoms < 0:
+            raise PypactOutOfRangeException("Number of atoms must be positive")
```

Comment:
Maybe a throw a new exception type, for example PypactUnphysicalValueException, or a similar?

Reply:
Added exception to exceptions.py and updated inputdata.py